SetUp/autoDefine/autoDefine.py

```diff
@@ -314,16 +314,30 @@ def inputBuilder( directories, options, keep_going ):
         defInp.close()
 
 
-        #Now copy the define input file over and run define with it. 
+        #Now move the define input file over and run define with it. 
         os.rename('def.input',dirs+'/def.input')
+        copyfile('templateCosmo.input',dirs+'/templateCosmo.input')
```

Comment:
Requiring a premade file makes the program more difficult to use, prefer to create it here, should mostly be writing \n and the dielectric constant